atlas/src/test/java/us/ihmc/atlas/obstacleCourseTests/AtlasFinalsWorldStairsTest.java

```diff
@@ -97,7 +97,8 @@ public void testWalkingUpStairs() throws BlockingSimulationRunner.SimulationExce
 
       success = success && simulationTestHelper.simulateNow(14.0);
 
-      simulationTestHelper.createBambooVideo(getSimpleRobotName(), 1);
+      // TODO GITHUB WORKFLOWS
+//      simulationTestHelper.createBambooVideo(getSimpleRobotName(), 1);
```

Comment:
How can we do this in a way that it doesn't screw up the workflow, but does make it easily searchable for everything where we want a video? I'm thinking renaming the method to "createTestVideo()" and slapping on a deprecated flag, such that the functionality can be easily added later without having to search for a b unch of commented out code.

Reply:
With it commented, you can still do a text search